spec/datadog/appsec/contrib/graphql/integration_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'datadog/appsec/spec_helper'
+require 'datadog/appsec/contrib/graphql/integration'
+
+RSpec.describe Datadog::AppSec::Contrib::GraphQL::Integration do
+  describe '.ast_node_classes_defined?' do
+    it 'returns true when all AST node classes are defined' do
+      expect(described_class.ast_node_classes_defined?).to eq(true)
```

Comment:
it is the same actually, I just thought that `be(true)` would be a bit more readable

Reply:
I personally type `expect(xxx).to be true` because it saves me two shift presses with pinky but as far as reading the code I don't see any difference between `be true` and `eq(true)`. Semantically I believe the two are identical.